PR#4356 Liquid Error

- [ ] Liquid Exception: Liquid syntax error (line 24): Unknown tag '_posts' in /home/runner/work/blog/blog/_posts/2026/08/2026-08-10-HegsethWifeAttendedMilitaryMeetingSensitiveInformation.md
bundler: failed to load command: jekyll (/home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/bin/jekyll)
/home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/liquid-4.0.4/lib/liquid/document.rb:23:in 'Liquid::Document#unknown_tag': Liquid syntax error (line 24): Unknown tag '_posts' (Liquid::SyntaxError)

```
Run bundle exec jekyll build --trace --incremental --baseurl "/blog"
  bundle exec jekyll build --trace --incremental --baseurl "/blog"
  shell: /usr/bin/bash -e {0}
  env:
    GITHUB_PAGES: true
    JEKYLL_ENV: production
    JEKYLL_GITHUB_TOKEN: 
    LOG_LEVEL: debug
To use retry middleware with Faraday v2.0+, install `faraday-retry` gem
Configuration file: /home/runner/work/blog/blog/_config.yml
            Source: /home/runner/work/blog/blog
       Destination: /home/runner/work/blog/blog/_site
 Incremental build: enabled
      Generating... 
          Skipping: _posts/2026/12/2026-12-25-FairyTaleOfBirth.md has a future date
          Skipping: _posts/2029/01/2029-01-20-ByeByeDonnie.md has a future date
  AI Related Posts: Creating cache [.ai_related_posts_cache.sqlite3]
  AI Related Posts: Fetch disabled. Using cached related posts data.
  AI Related Posts: Found 0 cached embeddings; skipped 1876 fetches
       Jekyll Feed: Generating feed for posts
         AutoPages: Disabled/Not configured in site.config.
        Pagination: ----------------------------
        Pagination: Page: blogpager.html
        Pagination:  Active configuration
        Pagination:   Enabled:          true
        Pagination:   Items per page:   15
        Pagination:   Permalink:        /page/:num/
        Pagination:   Title:            :title - page :num of :max
        Pagination:   Limit:            0
        Pagination:   Sort by:          date
        Pagination:   Sort reverse:     true
        Pagination:  Active Filters
        Pagination:   Collection:       posts
        Pagination:   Offset:           0
        Pagination:   Category:         [Not set]
        Pagination:   Tag:              [Not set]
        Pagination:   Locale:           [Not set]
        Pagination:  Filtering by: Category  1876 => 1876
        Pagination:  Filtering by: Tag       1876 => 1876
        Pagination:  Filtering by: Locale    1876 => 1876
        Pagination: Rolling through the date fields for all documents
        Pagination: Complete, processed 1 pagination page(s)
  
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/liquid-4.0.4/lib/liquid/document.rb:11:in 'block in Liquid::Document#parse'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/liquid-4.0.4/lib/liquid/block_body.rb:32:in 'Liquid::BlockBody#parse'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/liquid-4.0.4/lib/liquid/document.rb:10:in 'Liquid::Document#parse'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/liquid-4.0.4/lib/liquid/document.rb:5:in 'Liquid::Document.parse'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/liquid-4.0.4/lib/liquid/template.rb:130:in 'Liquid::Template#parse'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/liquid-4.0.4/lib/liquid/template.rb:114:in 'Liquid::Template.parse'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:13:in 'block in Jekyll::LiquidRenderer::File#parse'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:70:in 'Jekyll::LiquidRenderer::File#measure_time'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:12:in 'Jekyll::LiquidRenderer::File#parse'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/renderer.rb:124:in 'Jekyll::Renderer#render_liquid'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/renderer.rb:80:in 'Jekyll::Renderer#render_document'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/renderer.rb:63:in 'Jekyll::Renderer#run'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:572:in 'Jekyll::Site#render_regenerated'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:557:in 'block (2 levels) in Jekyll::Site#render_docs'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:556:in 'Array#each'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:556:in 'block in Jekyll::Site#render_docs'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:555:in 'Hash#each_value'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:555:in 'Jekyll::Site#render_docs'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:210:in 'Jekyll::Site#render'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:80:in 'Jekyll::Site#process'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:28:in 'Jekyll::Command.process_site'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:65:in 'Jekyll::Commands::Build.build'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:36:in 'Jekyll::Commands::Build.process'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in 'block in Jekyll::Command.process_with_graceful_fail'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in 'Array#each'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in 'Jekyll::Command.process_with_graceful_fail'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:18:in 'block (2 levels) in Jekyll::Commands::Build.init_with_program'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in 'block in Mercenary::Command#execute'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in 'Array#each'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in 'Mercenary::Command#execute'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in 'Mercenary::Program#go'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/mercenary-0.4.0/lib/mercenary.rb:21:in 'Mercenary.program'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/exe/jekyll:15:in '<top (required)>'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/lib/ruby/4.0.0/rubygems.rb:305:in 'Kernel#load'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/lib/ruby/4.0.0/rubygems.rb:305:in 'Gem.activate_and_load_bin_path'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/bin/jekyll:25:in '<top (required)>'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/lib/ruby/4.0.0/bundler/cli/exec.rb:61:in 'Kernel.load'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/lib/ruby/4.0.0/bundler/cli/exec.rb:61:in 'Bundler::CLI::Exec#kernel_load'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/lib/ruby/4.0.0/bundler/cli/exec.rb:24:in 'Bundler::CLI::Exec#run'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/lib/ruby/4.0.0/bundler/cli.rb:504:in 'Bundler::CLI#exec'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/lib/ruby/4.0.0/bundler/vendor/thor/lib/thor/command.rb:28:in 'Bundler::Thor::Command#run'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/lib/ruby/4.0.0/bundler/vendor/thor/lib/thor/invocation.rb:127:in 'Bundler::Thor::Invocation#invoke_command'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/lib/ruby/4.0.0/bundler/vendor/thor/lib/thor.rb:538:in 'Bundler::Thor.dispatch'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/lib/ruby/4.0.0/bundler/cli.rb:35:in 'Bundler::CLI.dispatch'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/lib/ruby/4.0.0/bundler/vendor/thor/lib/thor/base.rb:584:in 'Bundler::Thor::Base::ClassMethods#start'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/lib/ruby/4.0.0/bundler/cli.rb:29:in 'Bundler::CLI.start'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/lib/ruby/gems/4.0.0/gems/bundler-4.0.10/exe/bundle:28:in 'block in <top (required)>'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/lib/ruby/4.0.0/bundler/friendly_errors.rb:118:in 'Bundler.with_friendly_errors'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/lib/ruby/gems/4.0.0/gems/bundler-4.0.10/exe/bundle:20:in '<top (required)>'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/lib/ruby/4.0.0/rubygems.rb:305:in 'Kernel#load'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/lib/ruby/4.0.0/rubygems.rb:305:in 'Gem.activate_and_load_bin_path'
	from /opt/hostedtoolcache/Ruby/4.0.5/x64/bin/bundle:25:in '<main>'
Error: Process completed with exit code 1.
```

_Originally posted by @RalphHightower in https://github.com/RalphHightower/blog/issues/4356#issuecomment-5247081771_

--- a/_posts/2026/08/2026-08-10-HegsethWifeAttendedMilitaryMeetingSensitiveInformation.md
+++ b/_posts/2026/08/2026-08-10-HegsethWifeAttendedMilitaryMeetingSensitiveInformation.md
@@ -34,7 +34,7 @@ Just two months into [Pete Hegseth](https://www.defense.gov/About/Biographies/Bi
 
 > _[President](https://www.whitehouse.gov/) [Trump](https://www.donaldjtrump.com/) signed an executive order on Monday to form a new commission focused on improving the lives of military spouses and families and addressing the challenges they encounter._
 
-A month later, we learned that [Pete Hegseth](https://www.defense.gov/About/Biographies/Biography/Article/4040890/hon-pete-hegseth/) used his personal phone to send highly specific details about an imminent U.S. military strike to a 13-person [Signal](https://signal.org/) [group chat]({% _posts/2025/03/2025-03-25-YemenHouthiWarPlansTextedUnsecured.md %}), [which also included his wife](https://www.ms.now/rachel-maddow-show/maddowblog/intensifying-scandals-pete-hegseth-keeps-trumps-backing-long-rcna202315).
+A month later, we learned that [Pete Hegseth](https://www.defense.gov/About/Biographies/Biography/Article/4040890/hon-pete-hegseth/) used his personal phone to send highly specific details about an imminent U.S. military strike to a 13-person [Signal](https://signal.org/) [group chat]({% link _posts/2025/03/2025-03-25-YemenHouthiWarPlansTextedUnsecured.md %}), [which also included his wife](https://www.ms.now/rachel-maddow-show/maddowblog/intensifying-scandals-pete-hegseth-keeps-trumps-backing-long-rcna202315).
 
 A week later, [The Washington Post reported](https://www.washingtonpost.com/national-security/2025/04/30/jennifer-hegseth-pentagon-trump/):
 
@@ -59,8 +59,8 @@ If there’s ever been a defense secretary’s spouse with a portfolio like this
 - [The Hill covering Congress, Politics, Political Campaigns and Capitol Hill](https://thehill.com/)
 - [The Washington Post Breaking news and latest headlines, U.S. news, world news, and video The Washington Post](https://www.washingtonpost.com/)
 - [The Wall Street Journal (WSJ) Breaking News, Business, Financial & Economic News, World News and Video](https://www.wsj.com/)
-- [Shit You Can’t Make Up: Yemen Houthi Strike Planning Held On Unsecured Phones In Group Chat]({% _posts/2025/03/2025-03-25-YemenHouthiWarPlansTextedUnsecured.md %})
-- [Secretary of Defense, Peter Hegseth Is a Great Wingman For a Night 🌙 of Drinking 🍸 and Strip Clubs 👙. Call 867-5309]({% _posts/2025/04/2025-04-27-PeteHegsethCellphone8675309.md %})
+- [Shit You Can’t Make Up: Yemen Houthi Strike Planning Held On Unsecured Phones In Group Chat]({% llink _posts/2025/03/2025-03-25-YemenHouthiWarPlansTextedUnsecured.md %})
+- [Secretary of Defense, Peter Hegseth Is a Great Wingman For a Night 🌙 of Drinking 🍸 and Strip Clubs 👙. Call 867-5309]({% link _posts/2025/04/2025-04-27-PeteHegsethCellphone8675309.md %})
 - keywords
 - [Jennifer Rauchet Hegseth](https://www.linkedin.com/in/jennifer-hegseth-190b7890/)
 - [America First Policy Institute](https://www.americafirstpolicy.com/)

